src/secret/secret.tsx

```diff
@@ -0,0 +1,6 @@
+export function secretCheck(){
+    const queryString = window.location.search 
```

Comment:
Das aus dieser Funktion heraus auf window.location.search zugegriffen wird macht sie natürlich untestbar, weil man den Test korrumpiert sobald man window.location.search aus Testgründen modifiziert. Daher sollten Sie den String den Sie sich so in die Funktion holen als Parameter übergeben. Dann sollte es auch ein Leichtes sein, Tests für diese Funktion zu definieren.